### ISSUE TITLE
Anzahl der Tags begrenzen?

### ISSUE THREAD

**Issue by marcaltmann** [initiatived21/d21#217]
Hatten wir mal darüber gesprochen, dass Nutzer nur drei Tags vergeben dürfen?

Die Begrenzung ist übrigens mit der Select-Komponente, die wir im Moment verwenden, nicht so einfach möglich.

**Comment by annakress** [comment]
Ja, im Prinzip sollten es maximal 3 sein. Hab`s aber in der Validierung selbst nicht hinbekommen, das zu zählen. Ist nicht so super dringend, aber wir lassen es mal im Backlog, danke.
